docs/release_guidelines.md

```diff
@@ -1,23 +1,115 @@
+
+### Github UI
+
+From the Github UI, navigate to the list of [branches](https://github.com/red-hat-storage/ocs-ci/branches). From here you can click on new branch, selecting the `stable` branch as the base. Note that the branch creation process only works this way if the latest stable commit is the point we wish to branch off from. Otherwise we will need to create the branch using the CLI.
+
+### CLI
+
+First, ensure your local repository is up to date with the remote. Note that the following command is destructive so make sure you have any local changes stashed or committed before continuing.
+```
+git checkout master && git fetch --all && git reset --hard upstream/master
+```
+
+Checkout the stable branch
+```
+git checkout stable
+```
+
+Determine the starting point of the new release branch, for this you will need the git hash. You can find this using:
+```
+git log
+```
+
+Once you know the hash, we will create the new release branch using the hash from earlier as the base. Note that x.y is the major.minor version.
+```
+git checkout -b release-x.y 46568b9b4b8c2ceabad665b839655e0b19ab3634
+```
+
+Finally, push the branch to the remote.
+```
+git push upstream release-x.y
+```
+
+
+## Backporting Changes
+
+Most changes to the repository will end up being merged to master, propagage to the stable branch and evnetually end up on the next release branch. Critical bug fixes or important test cases may be selected to be backported to existing release branches. In order to backport changes to an existing release branch, we will need to take the following steps.
+
+### Creating a cherry-pick PR
+
+The first step in the process will be to checkout the master branch and ensure it is up to date, as well as fetch remote branches. Note that the following command is destructive so make sure you have any local changes stashed or committed before continuing.
+```
+git checkout master && git fetch --all && git reset --hard upstream/master
+```
+
+Before we can cherry-pick our change we will need to know the commit hash. You can find this using:
+```
+git log
+```
+
+Once we have the hash, we will create a local branch based off of the release branch we wish to backport our changes to.
+```
+git checkout -b release-4.8-cherry-pick-pr-4765 upstream/release-4.8
+```
+
+Then, we will cherry pick the commit to our branch using the hash.
+```
+git cherry-pick 46568b9b4b8c2ceabad665b839655e0b19ab3634
+```
+
+And push our branch to our fork of the repository.
+```
+git push origin release-4.8-cherry-pick-pr-4765
+```
+
+Once we have our branch pushed to our fork, we can then open a pull request from Github. Be sure to change the base branch to the target release branch (in our example this would be `release-4.8`).
```

Comment:
It might be worth to emphasize the **Be sure to change the base branch to the target release branch (in our example this would be `release-4.8`).**

Reply:
Done!